brands/shop/books.json

```diff
@@ -212,6 +212,16 @@
       "shop": "books"
     }
   },
+  "shop/books|Livraria Saraiva": {
```

Comment:
From their website and signage itit looks like this is just be Saraiva. Does that seem right? I don't see any references to the full name.

Reply:
Livraria translates into bookstore. Call it just Saraiva sounds kinda loose. Normaly when we go to the mall we see it as Livraria Saraiva, but I think that at the brand line I should change to just Saraiva. 

We can found some references to the full name at the wikipedia title, at a online phone list and a search at google for Saraiva we can see some malls calling it Livraria Saraiva.

But I don't have a problem changing it to Saraiva if you found that is better that way haha

Reply:
My own search for photos of the store shows mostly "Saraiva" signs.  I think it should be entered as "Saraiva" unless you know that it is regularly shown as "Livraria Saraiva" in the stores.